Add domain configuration for vnt941.is-a.dev

vnt941 opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE FOR YOUR PR TO BE ACCEPTED!
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->
<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/). <!-- Your JSON file is in the domains directory, the name is valid, etc. -->
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied template websites with minimal changes. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] My website is **not for commercial use**. <!-- Your website's purpose should not be to generate any form of revenue or profit. -->
- [x] I have provided contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your PR to be approved. -->

# Website Preview
<!-- Provide a link AND screenshot of your website below. -->
https://periodically-serves-calling-nova.trycloudflare.com/
![IMG_20251212_214127_324](https://github.com/user-attachments/assets/d2f09717-2e59-4ad7-948c-5b25a7dbb8d8)

vnt941 committed:
Add domain configuration for vnt941.is-a.dev

vnt941 commented:
Hello — this is a personal Emby media server reachable via Cloudflare Tunnel (temporary preview URL above). Please let me know if you need anything else. Thanks!

vnt941 committed:
Update domain owner and record configuration

FWEEaaaa1 commented on domains/vnt941.is-a.dev.json:
also remove the ".is-a.dev" from ur json name file

FWEEaaaa1 commented:
idk how this is dev related.. but we can leave this up to a maintainer's decision and find out.

vnt941 committed:
Add vnt941.json for domain configuration

This subdomain is for personal development and self-hosting experimentation. I am using a cloudflare tunnel and the project is non commercial only intended for personal development use

vnt941 committed:
Delete domains/vnt941.is-a.dev.json

FWEEaaaa1 commented:
<img width="558" height="157" alt="image" src="https://github.com/user-attachments/assets/98e50501-3e17-4ee7-b7b7-55fd11d53536" />

@vnt941 better close this pr sadly :sob: before a maintainer does it for ya
